ext/standaloneusers/ang/crmLogin/crmLogin.html

```diff
@@ -16,9 +16,21 @@
       <label for="passwordInput" class="form-label">{{:: ts('Password') }}</label>
       <input type="password" class="form-control crm-form-text" id="passwordInput" ng-model="$ctrl.password" required />
     </div>
+    <div class="input-wrapper" ng-if="$ctrl.mfa_remember > 0" style="padding-left: 1.5rem;">
+      <input style="margin: 0.2rem 0 0 -1.5rem;position:absolute;"
+             type="checkbox"
+             class="form-control crm-form-checkbox"
+             id="rememberInput"
+             ng-model="$ctrl.rememberMe"
+             ng-change="$ctrl.rememberMeChanged()" />
+      <div>
+        <label for="rememberInput" class="form-label">{{:: ts('Remember me') }}</label>
+        <div class="description">{{:: ts('Only use on a private device. Allows skipping multi-factor authentication on this device for %1 days.', {1: $ctrl.mfa_remember})}}</div>
```

Comment:
Minor UX nitpick: When I see "Remember me" on the username/password form, my brain assumes it's about staying logged-in.

For 2FA, the pattern I'm used to is for the form asking for that factor to have a checkbox (typically something like "Don't ask again on this device".)

The help text sufficiently describes the implemented behaviour, so I would not consider this a blocker.

Reply:
I said what? 🤔  Well, OK, let's turn this into a description of the design-question. It sounds like you're saying there are two choices:

1. Put the checkbox on the first page, and it will work with more MFA's.
2. Put the checkbox on the second page, and it will be more intuitive to each user.

If those are the choices, then I would actually lean toward the second -- because the checkbox is presented to users on a day-to-day basis (and I don't expect there to be too many MFA variants -- but maybe I don't understand the MFA variants). In any case, I don't have a strong opinion. 

If the choice is to go with (1) or wait three weeks for switch to (2), then I think is (1) fine. 😄 (But maybe switching is patchwelcome.)